pylintrc

```diff
@@ -59,7 +59,7 @@ disable=invalid-name,protected-access,fixme,too-many-branches,
         import-error,too-many-lines,too-many-instance-attributes,
         too-many-public-methods,duplicate-code,broad-except,
         redefined-builtin,anomalous-backslash-in-string,
-        missing-docstring,no-member,
+        missing-docstring,no-member,overlapping-except,
```

Comment:
This file is for testing pylint itself on CI. Ideally, any occurence of this new error should be fixed in the code base, instead of ignoring it.